examples/signature-agent-card-and-registry/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "signature-agent-card-and-registry"
+description = "Signature agent card and registry served over Cloudflare workers"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+readme.workspace = true
+homepage.workspace = true
+repository.workspace = true
+keywords.workspace = true
+categories.workspace = true
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+worker = { version = "0.7", features = ['http'] }
+worker-macros = { version = "0.7", features = ['http'] }
+web-bot-auth = { workspace = true }
+ed25519-dalek = { version = "2.2.0", features = ['rand_core'] }
+rand = "0.6.4"
```

Comment:
why `0.6.4`? the latest version is `0.9.2`.

if that's a wasm issue, you can add the following per their documentation https://docs.rs/getrandom/latest/getrandom/#webassembly-support

```
[target.'cfg(wasm32)'.dependencies]
getrandom = { version = "0.3.4", features = ["wasm_js"] }
```

Reply:
TIL. so either you use two pre-releases (rand@0.10.0 and ed25519-dalek@0.3.0) or you stay in these version. might be worth a comment